polaris-tokens/src/token-groups/breakpoints.ts

```diff
@@ -1,6 +1,11 @@
 import type {MetadataProperties} from '../types';
 
-export type BreakpointsAlias = 'xs' | 'sm' | 'md' | 'lg' | 'xl';
+// NOTE: Order is important here: smallest -> largest
+// Exporting as const means it will be typed as a Tuple instead of string[]
+export const breakpointAliases = ['xs', 'sm', 'md', 'lg', 'xl'] as const;
```

Comment:
```suggestion
export const breakpointsAliases = ['xs', 'sm', 'md', 'lg', 'xl'] as const;
```

Doesn't quite roll off the tongue as nice, but I think it's important to be consistent with token group prefixing 🙂